Max/Russia.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Maskirovka
+{
+
+	public class Russia : MonoBehaviour 
+	{
+		public List<Neighbour> neighbours;
+		private Color color;
+
+				
+		void Start () {
+			color = GetComponent<SpriteRenderer>().color;
+		}
+
+
+		public bool InvadeCountry(Country country) {
+			bool isNeighbour = neighbours.FindAll(x => x.neighbour ==  country ).Count > 0;
+			
+			if( isNeighbour == true ) {
+				country.Invaded(this);
+			}
+			return isNeighbour;
+		}
+
+
+		public Color GetColor() {
+			return color;
+		}
+
```

Comment:
The preffered rule for short single line if statements is to put it in one line.
Not a must, just preferred.